source/core/WorldBase.hpp

```diff
@@ -101,6 +102,44 @@ namespace cse491 {
       return *agent_set.back();
     }
 
+    /// @brief Build a new entity
+    /// @tparam PROPERTY_Ts Types for any properties to set at creation (automatic)
+    /// @param entity_name The name of this agent
+    /// @param properties Name/value pairs for any properties set at creation
+    /// @return A reference to the newly created entity
+    template <typename... PROPERTY_Ts>
+    Entity & AddEntity(std::string entity_name="None", PROPERTY_Ts... properties) {
```

Comment:
I would call this `AddItem` instead of `AddEntity` because it's going into `item_set`.  The only reason that I didn't create an `Item` class is because there was nothing to add beyond what was already in `Entity`, but maybe I should have for better code organization (and future updates to items?)